crystals/lexical_helper.F90

```diff
@@ -1628,7 +1628,42 @@ subroutine check_atom(image_text, ierror)
                                                   'TYPE    ', &
                                                   'FIRST   ', &
                                                   'UNTIL   ', &
-                                                  'MOVE    '/)
+                                                  'MOVE    ', &
+                                                  'RESTRAIN', &
+                                                  'ANGLES  ', &
+                                                  'VIBRATIO', &
+                                                  'U(IJ)''S ', &
+                                                  'PLANAR  ', &
+                                                  'SUM     ', &
+                                                  'AVERAGE ', &
+                                                  'LIMIT   ', &
+                                                  'NONBONDE', &
+                                                  'SAME    ', &
+                                                  'DELU    ', &
+                                                  'SIMU    ', &
+                                                  'A-VIBRAT', &
+                                                  'A-U(IJ)''', &
+                                                  'UXZBAR  ', &
+                                                  'UYZBAR  ', &
+                                                  'UZZBAR  ', &
+                                                  'UPERP   ', &
+                                                  'UALIGN  ', &
+                                                  'URIGU   ', &
+                                                  'ULIJ    ', &
+                                                  'UPLANE  ', &
+                                                  'UXXQIV  ', &
+                                                  'UXYQIV  ', &
+                                                  'UXZQIV  ', &
+                                                  'UYYQIV  ', &
+                                                  'UYZQIV  ', &
+                                                  'UZZQIV  ', &
+                                                  'UEIG    ', &
+                                                  'UQISO   ', &
+                                                  'UVOL    ', &
+                                                  'UEQIV   ', &
+                                                  'UTLS    ', &
+                                                  'XCHIV   ', &
```

Comment:
Is it not pretty much everything? That would render the subroutine check_atom useless